mayastor/src/subsys/config/opts.rs

```diff
@@ -188,6 +192,8 @@ impl Default for NvmfTcpTransportOpts {
             dif_insert_or_strip: false,
             max_aq_depth: 32,
             abort_timeout_sec: 1,
+            acceptor_poll_rate: try_from_env("NVMF_ACCEPTOR_POLL_RATE", 10_000),
+            zcopy: true,
```

Comment:
would it be worth adding another env variable for zcopy?

Reply:
Indeed, `NVMF_ZCOPY` it is then.